<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-5</y>
      <width>1051</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="choosePartWidget">
     <widget class="QTabWidget" name="choosePartTabs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>1051</width>
        <height>601</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="choosePartTab">
       <attribute name="title">
        <string>Choose Part</string>
       </attribute>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>40</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ramCombo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>230</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>380</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Power</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mainboard</string>
        </property>
       </widget>
       <widget class="QLabel" name="ramPrice">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>230</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>420</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sum :</string>
        </property>
       </widget>
       <widget class="QLabel" name="hddPrice">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>330</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="vgaPrice">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>180</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="sumPrice">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>420</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ssdPrice">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>280</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SSD</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadCombination">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>490</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
       <widget class="QComboBox" name="vgaCombo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cpuCombo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
       <widget class="QComboBox" name="hddCombo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>330</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>VGA</string>
        </property>
       </widget>
       <widget class="QComboBox" name="mbCombo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RAM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>HDD</string>
        </property>
       </widget>
       <widget class="QComboBox" name="powerCombo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>380</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="ssdCombo">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>280</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="saveCombination">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>490</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QLabel" name="cpuPrice">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>80</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="powerPrice">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>380</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="mbPrice">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>130</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="performanceTab">
       <attribute name="title">
        <string>Performance</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>481</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="chooseGameWidget">
     <widget class="QTabWidget" name="chooseGameTabs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>1051</width>
        <height>601</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="chooseGame">
       <attribute name="title">
        <string>Choose Game</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="quoteTab">
       <attribute name="title">
        <string>Quote</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>481</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
